src/appengine/handlers/jobs.py

```diff
@@ -98,6 +117,7 @@ def get(self):
             'templates': templates,
             'fieldValues': {
                 'csrf_token': form.generate_csrf_token(),
+                'fuzzers': fuzzers,
```

Comment:
seems like there's no need for the fuzzers variable. let's just use data_handler.get_all_fuzzer_names_including_children() inline here.

Reply:
done 👍